src/tdycore.c

```diff
@@ -1293,6 +1293,15 @@ PetscErrorCode TDySetDtimeForSNESSolver(TDy tdy, PetscReal dtime) {
   PetscFunctionReturn(0);
 }
 
+PetscErrorCode TDySetInitialCondition(TDy tdy, Vec initial) {
```

Comment:
Is it worth describing the form of the solution vector here under various circumstances, since this function is part of the library interface?

Reply:
Added a comment describing the function